app/livechat/client/views/app/tabbar/visitorInfo.js

```diff
@@ -324,6 +330,25 @@ Template.visitorInfo.events({
 
 		instance.action.set('transcript');
 	},
+
+	'click .on-hold'(event) {
+		event.preventDefault();
+		console.log('--on-hold manual button clicked');
+
+		return Meteor.call('livechat:placeChatOnHold', this.rid, function(error/* , result*/) {
```

Comment:
We need to use REST API methods instead of Meteor methods.

Reply:
@renatobecker Sorry I didn't get this

Do you mean create a new REST API on the backend and call that from the frontend? Example:
  [Backend API](https://github.com/RocketChat/Rocket.Chat/blob/978cd3a845b01a164e5b9fd7b82ad41d9a3ef4c8/ee/app/livechat-enterprise/server/api/monitors.js#L4)
  [FrontEnd call](https://github.com/RocketChat/Rocket.Chat/blob/978cd3a845b01a164e5b9fd7b82ad41d9a3ef4c8/ee/client/omnichannel/monitors/MonitorsPage.js#L48)

Reply:
Exaclty.